src/components/navbar/togglemenu/ToggleMenu.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components'
```

Comment:
I suggest you have your styled-components in another file. So it's external, and can be imported to your other React files

Reply:
Really?? ..or probably I don't get what you 're saying.
I am a little confused. 
Are you talking about the hamburger button? To keep it in a separate file, right? 
If so, yes I will do it. I just want to have an idea first, since I need to get used my brain/mind to moving all these components without losing the logic.
is that what you meant?

Reply:
ok done!! :)